### ISSUE TITLE
[Enhancement] Visual config GUI

### ISSUE THREAD

**Issue by Madis0** [Low-Drag-MC/Shimmer#71]
I'm aware that the mod adds `/shimmer`, but it is a clientside mod, so most users would still expect a visual config GUI in the mod menu. 

Maybe have both, with the visual one only working when a specific library (e.g. Cloth Config) is present? 

Edit: apparently the mod already uses Cloth Config, so a visual GUI could already be added by that.

**Comment by KawaiiBakemono** [comment]
I agree with this, the fact that there was no GUI options threw me through a loop, I searched for hotkeys to open the menus shown on the readme, looked for the config folder (it was empty), and THEN figured out that there's a command to access it all. It's very unintuitive and without knowing it exists, most users would be lost. There is no mention of there needing a command to access the settings. Furthermore, none of the options on these screens are explained. What's the difference between "add" and "apply"? What does "export" do? "Import color" doesn't work for me because I cannot for the life of me figure out how the eyedropper tool works, another thing that should be addressed. 

TLDR: There needs to be documentation for this mod, otherwise no one will know how to use it. This is an honest mistake, I guarantee the mod creator knows exactly how everything works, but they shouldn't expect that everyone else can figure out on their own. Config options within Mod Menu would be great!
